### DISCUSSION TITLE
is this banable from discord ?? :(

### DISCUSSION THREAD

**Discussion by nischay876** [Tyrrrz/DiscordChatExporter#726] [Q&A]
asking this because my account was clear clean and after i export some chat in 3 days my accound got banned

**Comment by Tyrrrz** [comment]
Unlikely, most people use it without issues. But there's never any guarantee.

**Reply by codemonies** [reply]
Not sure why their team is adamant about not taking responsibility, but yes, Discord hates it.

They disabled my account instantly after joining their Discord support group. I tried to warn others over [here](https://github.com/Tyrrrz/DiscordChatExporter/discussions/707).

On appeal, I told them I found out about DiscordChatExporter from their own community forums, and they restored it.

So, clearly, this was the culprit.

**Reply by 96-LB** [reply]
The Discord you linked is a general hub for multiple projects/applications, and not dedicated solely to DiscordChatExporter. Furthermore, DiscordChatExporter is perfectly within the Terms of Service if you use a bot token as recommended. It's highly unlikely that you would have your account disabled simply for the act of joining the server—otherwise, there would probably be some issues for the 300 people currently on it.

The truth is that there have been very few reports of being banned for using DiscordChatExporter, but if you use a user token there is always a risk. Nobody is trying to hide this; it's not like DCE is trying to get anyone banned. There does seem to be an uptick in bans for self-automation lately, so as usual it is recommended to use a bot account.

If your ban was successfully appealed, it's likely you did nothing wrong in the first place, so I understand why you'd be upset about the whole ordeal. I don't know if pinning it on DCE is correct, however.

**Reply by codemonies** [reply]
I have nothing to gain by being dishonest. I joined the group to ask some questions before using the application. It instantly logged me out. When I tried logging back into Discord, it said the account is disabled. I appealed the ban since the trigger was obvious. I mentioned in the appeal that I had simply joined the group based on a mention of it in their own community forum, and they reinstated the account.

If joining the group was not the trigger, why would I be auto-banned as soon as I joined? As well, why would they reinstate the account when I specifically mentioned that I had not yet used the tool and simply joined the group for support? I obviously hit the nail on the hand in terms of the culprit in my appeal.

**Reply by 96-LB** [reply]
I'm not trying to accuse you of lying. But the fact of the matter is that the server has had hundreds of people join without issue and (as far as I know) has not received any notification from Discord, so it's unlikely that it's being used to "catch" people who violate the TOS with the tool. There's likely something else at play also, but I can't pretend to have any idea how Discord flags accounts behind the scenes. Either way, you're right that Discord does not want people breaking the terms of service, and yes they can take action if you do so.

**Reply by Chrombastic** [reply]
> asking this because my account was clear clean and after i export some chat in 3 days my accound got banned

were you able to get your account unbanned?

**Reply by codemonies** [reply]
> > asking this because my account was clear clean and after i export some chat in 3 days my accound got banned
> 
> were you able to get your account unbanned?

Yes, I had to appeal the ban. They restored it after a few days.

**Reply by dchilton** [reply]
Is it against discord terms to export data?

**Reply by codemonies** [reply]
> Is it against discord terms to export data?

Yes, automation tools are against their ToS.

**Reply by 96-LB** [reply]
@dchilton It's against terms of service to automate a user account. Exporting with a bot account is allowed and will not get you banned.

**Reply by nischay876** [reply]
> > asking this because my account was clear clean and after i export some chat in 3 days my accound got banned
> 
> were you able to get your account unbanned?

no

**Reply by tomekit** [reply]
Can similar functionality as `DiscordCharExporter` has, be achieved with bot account that exports things?
